ONSdigital/sdc-int-rh-ui
# Socint 251 update get uac response object

# Motivation and Context
Have updated code based on changes made to object as part of SOCINT-246

# What has changed
Changes have been made to the object associated with the new UniqueAccessCodeDTO in the RH SVC.

# How to test?
1. Run unit tests
2. Run acceptance tests
3. In the RH UI enter a know UAC and request survey, click yes to address and this should redirect to EQ.

# Links
Ticket https://collaborate2.ons.gov.uk/jira/browse/SOCINT-251
Associated ticket https://collaborate2.ons.gov.uk/jira/browse/SOCINT-246
